1080i/Includes_View_52_Showcase.xml

```diff
@@ -114,14 +115,15 @@
                         <param name="right" value="30" />
                     </include>
                     <control type="grouplist">
-                        <top>451.435</top>
+                        <top>440.435</top>
```

Comment:
I'm not too keen on this change to the spacing as it will now be inconsistent with other views. I'm not sure where else you would get the extra pixels from though...